src/libraries/System.Net.Sockets/tests/FunctionalTests/SocketTestHelper.cs

```diff
@@ -135,7 +135,10 @@ public override async Task<(Socket socket, byte[] buffer)> AcceptAsync(Socket s,
             return (socket, buffer);
         }
         public override Task<Socket> AcceptAsync(Socket s, Socket acceptSocket) =>
-            Task.Factory.FromAsync(s.BeginAccept, s.EndAccept, acceptSocket, 0, null);
+            Task.Factory.FromAsync(
+                (callback, state) => s.BeginAccept(acceptSocket, 0, callback, state),
+                result => s.EndAccept(out _, out _, result),
+                null);
```

Comment:
Are there any tests that were disabled on unix and should now be enabled?  If not, are we missing tests?

Reply:
I forgot to reenable the tests on Linux. Will update.